react-native-purchases-ui/ios/RNCustomerCenter.m

```diff
@@ -49,7 +50,17 @@ - (void)initializeCustomerCenter {
 // MARK: -
 
 - (NSArray<NSString *> *)supportedEvents {
```

Comment:
Hmm I didn't find it from a quick search, what was this used for?

Reply:
This is needed because this is the content of `RCTEventEmitter.m`. It's required to implement it

```
- (NSArray<NSString *> *)supportedEvents
{
  NSString *message =
      [NSString stringWithFormat:@"%@ must implement the supportedEvents method", NSStringFromClass(self.class)];
  [self _log:message];
  return nil;
}
```